setup.py

```diff
@@ -156,7 +156,7 @@ def get_version() -> str:
           'verovio.data.Petaluma': os.listdir('./data/Petaluma'),
           'verovio.data.text': os.listdir('./data/text'),
       },
```

Comment:
We should add the `py.typed` here, see https://peps.python.org/pep-0561/#packaging-type-information.

Reply:
I think it gets copied here:

```py
      package_dir={'verovio': './bindings/python',
                   'verovio.data': './data'},
```

Reply:
I've added it to the package_data object... not sure what it does, but it's there.